lobby/src/main/java/org/triplea/lobby/server/LobbyGameController.java

```diff
@@ -60,26 +59,15 @@ private void connectionLost(final INode to) {
 
   @Override
   public void postGame(final GUID gameId, final GameDescription description) {
-    final INode from = MessageContext.getSender();
-    assertCorrectHost(description, from);
```

Comment:
I just realized that even though this works, it's not safe to do that without any check at all.
I think the primary reason for this check was to ensure nobody could add an arbitrary amount of games to the lobby, or modify existing game entries.
We need another way to make sure there's only one entry per existing game.